src/java/main/org/apache/zookeeper/server/SessionTracker.java

```diff
@@ -47,21 +47,20 @@
     long createSession(int sessionTimeout);
 
     /**
-     * Add a global session to those being tracked.
+     * Track the session expire, not add to ZkDb.
      * @param id sessionId
      * @param to sessionTimeout
      * @return whether the session was newly added (if false, already existed)
      */
-    boolean addGlobalSession(long id, int to);
+    boolean trackSession(long id, int to);
```

Comment:
Is `trackSession` now only tracking global sessions?

Reply:
Local session tracker will start track session when create the session.